### PR TITLE
The Documentation Update (again)

### PR DESCRIPTION
**Pull request by lakshayrohila**
Once again, I present to you the same old documentation update that I presented to you a few months back (might be a little bit more modified, but I don't remember doing much). I had thought that I'll complete it and then present it, but due to some time limitations, I don't think I'll be able to complete all the parts of it - at least not this year.

I think it would be unwise to keep all the done work with me when I can no longer work on it. Sorry for the delay, but this is the final draft of the documentation I could come up with. I may later on like to work with Puppy, but currently, I wouldn't be able to.

@dimkr, @01micko, @peabee Please review. Here is Jammy64 built with these changes (at commit af449c6): [link](https://github.com/lakshayrohila/woof-CE/releases/tag/jammypup64ce-10.0-docs-test)

Also, if you want me not to directly open a PR to woof-CE for this and if someone else would like to finish the work, I am open to opening a PR to their woof-CE fork instead.

### CODE REVIEW

File: woof-code/rootfs-petbuilds/connman-gtk/petbuild [hunk 1]
```diff
@@ -9,5 +9,5 @@ build() {
     patch -p1 < ../icons.patch
     meson --buildtype=release --prefix=/usr -Duse_status_icon=false build
     ninja -C build install
-    sed -e 's/^Categories=.*/Categories=Network;/' -e 's~^Icon=.*~Icon=network_connect~' -i /usr/share/applications/connman-gtk.desktop
-}
\ No newline at end of file
+    rm -rf /usr/share/applications
```
Comment (dimkr) [inline woof-code/rootfs-petbuilds/connman-gtk/petbuild]: Why this change?

File: woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm [hunk 2]
```diff
@@ -1,11 +1,13 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>HOWTO Internet</title>
+	<title>HOWTO Internet by Dialup</title>
```
Comment (dimkr) [inline woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm]: Is this page still relevant and useful?

**Comment by dimkr** [comment]
Are the images smaller if converted to .png plus `optipng -o7` plus `advpng -z4`?

File: woof-code/rootfs-petbuilds/connman-gtk/petbuild (hunk 1 above)
Comment (lakshayrohila) [reply woof-code/rootfs-petbuilds/connman-gtk/petbuild]: Sorry, mistake while resolving git merge.

File: woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm (hunk 2 above)
Comment (lakshayrohila) [reply woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm]: IDK. It was already there so I put it in, though it hasn't been updated.

**Comment by lakshayrohila** [comment]
> Are the images smaller if converted to .png plus `optipng -o7` plus `advpng -z4`?

@dimkr, I am not sure about that. How can I test? For all the images I have put in, I have tried reducing their sizes the most GUI image compressors could do. Few images were smaller in png while some in jpg.

**Comment by dimkr** [comment]
> How can I test?

Save as png, `optiong -o7`, `advpng -z4`, then check size.

**Comment by lakshayrohila** [comment]
> Save as png, `optiong -o7`, `advpng -z4`, then check size.

@dimkr, those which were jpg remain multiple times smaller than png gotten by those two commands (I even ran the more optimization one `optipng -o7 -zm1-9` as well). I also optimized other PNGs in the doc directory which could be optimized.

Please tell me any other command you think is needed to optimize the JPGs.

**Comment by dimkr** [comment]
AFAIK there's not much you can do to reduce JPG size. For big images JPG can be much smaller than PNG.

File: woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm (hunk 2 above)
Comment (lakshayrohila) [reply woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm]: @dimkr, do you have any idea what to do with this file? (or @01micko, @peabee)

Comment (lakshayrohila) [reply woof-code/rootfs-skeleton/usr/share/doc/HOWTO_dialup.htm]: I think this file is somewhat useless according to current internet standards. I don't think anyone uses dial-up these days. So, I am deciding to delete this file.
